composer.json

```diff
@@ -14,11 +14,11 @@
     ],
     "require": {
         "robrichards/xmlseclibs": "^3.0",
-        "php": "^7.2"
+        "php": "^7.2 || ^8.0"
```

Comment:
I saw in .travis.yml that tests are run with PHP 7.4 and above, but the library requires PHP 7.2. We should either use php 7.2 in travis' tests or bump the library's requirement to PHP 7.4 for consistency. 
Personally I'd use 7.4 since it's not EOL yet (EOL is the 28th of November 2022)